### PR TITLE
added in a simple page for triggering the various data mgmt actions

### PR DESCRIPTION
**Pull request by techsmyth**
Creates a simple html page at the data-management location that can be used to trigger the data management actions supported by the server

### CODE REVIEW

**Review by valentinyanakiev — CHANGES_REQUESTED** [review]
Doesn't work - clicking the buttons returns:
- {"statusCode":404,"message":"Cannot GET /reset-db?","error":"Not Found"}
- {"statusCode":404,"message":"Cannot GET /empty-ecoverse?","error":"Not Found"}
- {"statusCode":404,"message":"Cannot GET /seed-data?","error":"Not Found"}

File: src/utils/data-management/data-management.service.ts [hunk 1]
```diff
@@ -150,20 +158,46 @@ export class DataManagementService {
...
+  async populatePageContent(message: string): Promise<string> {
+    const ecoverse = await this.ecoverseService.getEcoverse();
+    const content = `<!DOCTYPE html>
+    <html>
+    <body>
+    <h1>Cherrytwist Data Management Utility</h1>
+    <h2>Ecoverse: <i>${ecoverse.name}</i></H2>
+    <p>
+    <b>Messages:</b>${message}</p>
+    <form action="./reset-db">
```
Comment (atstoyanov) [inline src/utils/data-management/data-management.service.ts]: Does not work. change it to "/data-management/reset-db"

File: src/utils/data-management/data-management.service.ts [hunk 2]
```diff
@@ -150,20 +158,46 @@ export class DataManagementService {
...
+    <input type="submit" value="Reset DB" />
+    </form>
+    <p><form action="./empty-ecoverse">
```
Comment (atstoyanov) [inline src/utils/data-management/data-management.service.ts]: Does not work. change it to "/data-management/empty-ecoverse"

File: src/utils/data-management/data-management.service.ts [hunk 3]
```diff
@@ -150,20 +158,46 @@ export class DataManagementService {
...
+    <input type="submit" value="Reset Ecoverse" />
+    </form></p>
+    <p><form action="./seed-data">
```
Comment (atstoyanov) [inline src/utils/data-management/data-management.service.ts]: Does not work. change it to "/data-management/seed-data"

**Review by atstojanov — CHANGES_REQUESTED** [review]

**Review by valentinyanakiev — APPROVED** [review]
👍

**Review by valentinyanakiev — APPROVED** [review]

**Pull request merged**